.github/workflows/release.yml

```diff
@@ -67,47 +44,6 @@ jobs:
       uses: Ilshidur/action-slack@2.0.0
```

Comment:
It is extremely puzzling where this gets the correct channel from. Which I believe to be `sonarqube-build`.